### ISSUE TITLE
Fairsharing: compare and add

### ISSUE THREAD

**Issue by veitveit** [edamontology/edamontology#531]
https://fairsharing.org/standards/?q=&selected_facets=type_exact:model/format&view=table

To be done:
export table and compare for coverage in EDAM formats
Fill in gaps
Think about information in Fairsharing that is usable for EDAM.

**Comment by joncison** [comment]
hey @veitveit 

made (or planning) any progress on this ?

**Comment by veitveit** [comment]
Yes, you will soon get a list for updates. I am also compiling a list of new terms but this will still take some time

**Comment by joncison** [comment]
Cool, thanks !

**Comment by veitveit** [comment]
@joncison These are updates for the terms I found in both:
https://docs.google.com/spreadsheets/d/1IIdkgs7yQ1guTgbLJb3nOcA4Zb1DIYGoXZxmlSNlDug/edit?usp=sharing

**Comment by joncison** [comment]
Excellent ... the "updates" I can probably do manually.  The "new terms" (once it's finalised) I think would benefit from a script to generate the RDF/XML.   Let me know once the "updates" are ready to go.

**Comment by veitveit** [comment]
The updates should be ready
